src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -268,6 +272,9 @@ def load_arguments(self, _):
         c.argument('disk_caching', nargs='*', help="Use singular value to apply across, or specify individual disks, e.g. 'os=ReadWrite 0=None 1=ReadOnly' should enable update os disk and 2 data disks")
 
     with self.argument_context('vm create') as c:
+        VMPriorityTypes = self.get_models('VirtualMachinePriorityTypes', resource_type=ResourceType.MGMT_COMPUTE)
+        VirtualMachineEvictionPolicyTypes = self.get_models('VirtualMachineEvictionPolicyTypes',
```

Comment:
make sure this won't throw under older api-version that the `VirtualMachinePriorityTypes` is unavailable

Reply:
I think we are running some old tests, so that should be covered

Reply:
Agree with Zim.
It's better to know the exact behavior when can't load model from a specific API version.

Reply:
Add a if statement, then get models
```
if self.supported_api_version(min_api='2019-03-01', resource_type=ResourceType.MGMT_COMPUTE):
```